providers/base/units/eeprom/jobs.pxu

```diff
@@ -0,0 +1,39 @@
+id: eeprom/read-write
+_summary: Test EEPROM read and write functions for the system. 
+plugin: shell
+user: root
+category_id: eeprom 
+flags: also-after-suspend
+estimated_duration: 5
+imports: from com.canonical.plainbox import manifest
+requires: manifest.has_eeprom == 'True'
+command:
+  eeprom_path=$(find /sys/ -name eeprom)
```

Comment:
This unnecessarily broad, and will cause some devices like dragonboard to crash the system.
I recommend going for a much narrower search path.

Reply:
I will change the path to "/sys/devices/platform/soc/".